peering/forms.py

```diff
@@ -707,6 +707,11 @@ class RouterForm(BootstrapMixin, forms.ModelForm):
     )
     comments = CommentField()
     tags = TagField(required=False)
+    
+    napalm_username = forms.CharField(required=False, label="Username")
+    napalm_password = forms.CharField(required=False, label="Password")
+    napalm_timeout = forms.IntegerField(label="Timeout in seconds", initial=0)
```

Comment:
Mark this field as not required, set its default value to 30 seconds like defined in the model. Call it just `Timeout` and set a helper text to indicate what it is for and what is its unit.